client/test/TestFluid.res

```diff
@@ -4069,11 +4069,13 @@ let run = () => {
       ()
     })
 
-    describe("create", () => {
-      t("create tuple", b, ~pos=0, ins("("), "(~___,___)")
-      t("create and fill in tuple", b, ~pos=0, insMany(list{"(", "1", ",", "2", ")"}), "(1,2)~")
-      ()
-    })
+    if defaultTestProps.settings.allowTuples {
```

Comment:
Is there a reason for this not to be default true?

Reply:
Hmm. Walking through this:

Currently we have:
```
let defaultTestProps: FluidTypes.Props.t = {
  functions: Functions.empty |> Functions.setBuiltins(defaultTestFunctions, defaultFunctionsProps),
  settings: FluidTypes.FluidSettings.default,
}

let defaultTestModel = {
  ...AppTypes.Model.default,
  functions: defaultTestProps.functions,
  analyses: ...
  fluidState: defaultTestState,
}
```

in `FluidTestData.res`, where these default values are stored.

Setting `settings` to `{ allowTuples: true }` here seems insufficient - those `defaultTestProps` aren't used in the context needed by this? I'm guessing `propsFromModel` is being called at some level down the chain...

That said, if I add this mess to `defaultTestModel`, the test passes:
```
settings: {
    ...AppTypes.Model.default.settings,
    contributingSettings: {
      ...AppTypes.Model.default.settings.contributingSettings,
      inProgressFeatures: {allowTuples: true},
    },
  },
```

but that's pretty ugly...

Is that the path you were thinking, or am I misinterpreting this question?
Hm, maybe `defaultTestProps` just isn't being passed into enough places.

Relatedly, it'd be nice(r) to 'set experimental features' per-test somewhere? :thinking:

Reply:
Anyway, I'm going to merge as-is and we can follow up here. Eventually (hopefully soonish) the setting will fade out anyway.